### PR TITLE
Testing/db utils

### PR DESCRIPTION
**Pull request by dutterbutter**
**Changes made in this pull request:**

In order to run the test set up SHYFT_ENV=test > cd shyftDb > run go test. To see the test coverage run go test --cover

- Unit testing for shyft_db_utils 
- Added testdb
- Resolved conflicts with shyftBlockExplorer

### CODE REVIEW

File: shyft-cli/postgres_setup/create_tables.psql [hunk 1]
```diff
@@ -38,7 +38,4 @@ CREATE TABLE IF NOT EXISTS accounts (
   balance numeric,
   txCountAccount numeric
 );
-
-CREATE TABLE IF NOT EXISTS contracts (
-  txHash text
-);
+shyft-cli/postgres_setup/create_tables.psql
```
Comment (GregTheGreek) [inline shyft-cli/postgres_setup/create_tables.psql]: This isn't psql

File: shyft-cli/postgres_setup/drop_tables.psql [hunk 2]
```diff
@@ -1,4 +1,4 @@
 DROP TABLE txs;
 DROP TABLE blocks;
 DROP TABLE accounts;
-DROP TABLE contracts;
+shyft-cli/postgres_setup/drop_tables.psql
```
Comment (GregTheGreek) [inline shyft-cli/postgres_setup/drop_tables.psql]: This isn't psql

Comment (dutterbutter) [reply shyft-cli/postgres_setup/drop_tables.psql]: changes made in latest commit

File: shyft-cli/postgres_setup/create_tables.psql (hunk 1 above)
Comment (dutterbutter) [reply shyft-cli/postgres_setup/create_tables.psql]: changes made in latest commit

File: shyftDb/postgres_setup_test/create_tables_test.psql [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  isContract bool,
+  age timestamp,
+  data bytea
+);
+
+CREATE TABLE IF NOT EXISTS accounts (
+  addr text primary key unique,
+  balance numeric,
+  txCountAccount numeric
+);
```
Comment (GregTheGreek) [inline shyftDb/postgres_setup_test/create_tables_test.psql]: Can we not use the other .psql files to do this? That way we don't maintain two sets of identical code?

**Pull request merged**